app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class FeedbacksController < ApplicationController
+  FEEDBACK_TYPES = %w[freetext satisfaction].freeze
+
+  def create
+    if params[:type].nil? || !FEEDBACK_TYPES.include?(params[:type])
+      raise "Feedback type needs to be specified"
+    end
+
+    @form = params[:type].include?("satisfaction") ? satisfaction_model : freetext_model
```

Comment:
I'm not sure `include?` is the ideal check here - we're checking to see if params[:type] _equals_ satisfaction. It works out as practically the same thing, because any string that includes satisfaction but doesn't equal it would raise an error on the previous line, but for clarity, an equals might be better.